source/endpoint-reference/endpoint-folders.rst

```diff
@@ -201,10 +197,30 @@ Trash
 
 ``/datasets/{id}/folders/trash/``
 
-A special folder, the folders' trash exists to store folders after they get
-deleted.
+The trash folder is a special-purpose folder that lives outside the
+regular variable folders tree.
+
+You can use the trash folder to "soft-delete" variables and folders.
+
+Subfolders and variables inside the trash folder are only visible to
+dataset editors.
+
+Performing a DELETE request to the trash endpoint will empty all items
+from the trash, hard-deleting the folders, variables, and related data.
+
+Items in the trash folder may also be automatically hard-deleted after
```

Comment:
This isn't quite true yet, is it?

Reply:
No, but that's why I used the word "may". I added it as a warning so that we don't need a deprecation cycle when we add that cron. People should be aware from the start that the trash folder is ephemeral.